dummy_robot/dummy_map_server/src/dummy_map_server.cpp

```diff
@@ -16,13 +16,11 @@
 #include <iostream>
 #include <memory>
 
-#include "rcl/rcl.h"
 #include "rclcpp/rclcpp.hpp"
+#include "rcl/rcl.h"
```

Comment:
Nit: rcl before rclcpp was good. nav_msgs is the include I was referring to (that should be before the rcl* ones)